worlds/Arena.wbt

```diff
@@ -206,13 +206,13 @@ DEF TOKEN_CTRL Robot {
       zoneColour 1 1 0
     }
     SBTokenHarness {
-      tokenName "ZONE_0_17"
+      tokenName "ZONE_1_17"
       emitterName "ZONE_0_17 Emitter"
```

Comment:
The emitter name needs to change here too.

Reply:
Fixed